### ISSUE TITLE
Codes with leading zeros fail verification

### ISSUE THREAD

**Issue by spikyjt** [Spomky-Labs/otphp#27]
Google Authenticator generates codes that are always 6 digits long, so are zero padded strings, rather than integers. However, this library generates integers, so they are not always the number of digits specified. This causes problems when verifying the submitted codes.

My solution was to override the `at()` method as follows:

```php
    public function at( $timestamp ) {
        return sprintf( '%06d', parent::at( $timestamp ) );
    }
```

**Comment by Spomky** [comment]
Hi,

This bug has been fixed in the develop branch, see f2adba427a0f5b1910870723afa442ffb6afa9e4.

Can you confirm the problem is solved?
Then, I will tag a new revision.

**Comment by Spomky** [comment]
Tagged as `v5.0.0`
